### PR TITLE
Fixed typo in error message

### PR DESCRIPTION
**Pull request by DevonFulcher**
### Description

Addresses this issue https://dbt-labs.slack.com/archives/C05K4R7KZ5Z/p1702571237772439. No dev release should be necessary for this.

### CODE REVIEW

**Comment by DevonFulcher** [comment]
Please merge this after approval

**Review by courtneyholcomb — APPROVED** [review]

**Pull request merged**
